lib/os_mon/src/memsup.erl

```diff
@@ -246,7 +246,7 @@ get_system_memory_data() ->
 -doc """
 Returns the time interval, in milliseconds, for the periodic memory check.
 """.
--spec get_check_interval() -> Milliseconds :: integer().
```

Comment:
Probably ok, anyone think differently ?

Reply:
That is my point. If it is documented, should it not have a spec?